newIDE/app/src/EventsFunctionsExtensionEditor/index.js

```diff
@@ -219,18 +219,18 @@ export default class EventsFunctionsExtensionEditor extends React.Component<
     newName: string,
     done: boolean => void
   ) => {
-    if (!gd.Project.validateObjectName(newName)) {
+    if (!gd.Project.validateName(newName)) {
       showWarningBox(
         i18n._(
-          t`This name contains forbidden characters: please only use alphanumeric characters (0-9, a-z) and underscores in your function name.`
+          t`This name is invalid. Only use alphanumeric characters (0-9, a-z) and underscores. Digits are not allowed as the first character.`
         )
       );
       return;
     }
     if (isExtensionLifecycleEventsFunction(newName)) {
       showWarningBox(
         i18n._(
-          t`This name is reserved for a lifecycle function of the extension. Choose another name for your function.`
```

Comment:
Careful, you've changed the wrong message here, it's a regression

Reply:
Mistake on my part. Updated the wrong message here.

Reply:
I fixed this for you, no need to push anything